tfrog-motordriver/communication.c

```diff
@@ -1696,6 +1697,9 @@ int command_analyze(unsigned char* data, int len)
       case PARAM_watch_dog_limit:
         driver_param.watchdog_limit = i.integer;
         break;
+      case PARAM_heartbeat:
+        heartbeat_timeout = 0;
```

Comment:
Can it be automatically resumed after the timeout?

Reply:
done

Reply:
Sorry, it would be better not to be automatic.
It's better to count heartbeat only if `> SERVO_LEVEL_STOP`.

Reply:
No worries. That makes much more sense now. ;)